civis/io/_files.py

```diff
@@ -470,6 +470,10 @@ def file_to_dataframe(file_id, compression='infer', client=None,
     client = APIClient() if client is None else client
     file_info = client.files.get(file_id)
     file_url = file_info.file_url
+    if not file_url:
+        raise ValueError(
+            "File url does not exist for file {}. File may be expired.".format(
+                file_id))
```

Comment:
Do we want to make this a general check for the other file functions as well (like `file_to_json`)? We're not running into issues with it right now but it might make sense to have them throw consistent errors

Reply:
Good point! We definitely should.

Reply:
As it turns out, the other file functions do have a check like this, and `file_to_dataframe` was the exception because it doesn't call `civis_to_file`. I updated the exception to match the pre-existing one in the other functions.

Reply:
Ah got it.